pages/organizations.js

```diff
@@ -0,0 +1,223 @@
+import React, { useEffect, useState } from 'react'
+import { connect } from 'unistore/react'
+import {
+  Menu,
+  MenuList,
+  MenuButton,
+  MenuItem
+} from '@reach/menu-button'
+
+import Table from '../components/common/Table'
+import Link from '../components/Link'
+import NotLoggedIn from '../components/NotLoggedIn'
+import { actions } from '../lib/store'
+import { LoadingState } from '../components/common/LoadingState'
+import AdminUsersSearch from '../components/admin/AdminUsersSearch'
+
+export function ManageOrg (props) {
+  const { getAuthenticatedUser, authenticatedUser, getOrganization, organization, getUserList, users } = props
+  const [loading, setLoading] = useState(true)
+  const [members, setMembers] = useState(null)
+  const [warning, setWarning] = useState(null)
+
+  // On load get the user
+  useEffect(() => {
+    getAuthenticatedUser()
+      .then(() => getOrganization())
+      .then(() => setLoading(false))
+  }, [])
+
+  useEffect(() => {
+    if (organization) {
+      const memberList = organization.organization.owners.concat(organization.organization.managers)
+      const memberIds = memberList.map(member => member.osm_id.toString())
+      getUserList(memberIds)
+    }
+  }, [organization])
+
+  useEffect(() => {
+    if (users) {
+      // Moves roles into state
+      let membersWithRoles = users.map(user => {
+        return organization.organization.owners.find(owner => owner.osm_id === user.osm_id) ? (
+          { ...user, role: 'owner' }
+        ) : (
+          { ...user, role: 'manager' }
+        )
+      })
+      setMembers(checkForSingleOwner(membersWithRoles))
+    }
+  }, [users])
```

Comment:
This useEffect also depends on the `organization` variable from the closure. It may be stale unless you list it as well in the dependency like: `[users, organization]`.

Reply:
(probably unrelated to the observed problem though)